ibis/backends/dask/convert.py

```diff
@@ -74,7 +74,7 @@ def convert_Date(cls, s, dtype, pandas_type):
         else:
             s = dd.to_datetime(s)
 
-        return s.dt.normalize()
```

Comment:
I think might want to keep this, but not 100% sure. In theory once you're in this function you shouldn't need to normalize ... in theory :)